docs/controls/InAppNotification.md

```diff
@@ -0,0 +1,129 @@
+# In App Notification 
+
+The *In App Notification* control offers the ability to show local notifications in your application.
+
+## Syntax
+
+The control should be placed where you want your notification to be displayed in the page, generally in the root grid.
```

Comment:
@Odonno  I think we should either say it in Doc or we should a default `Canvas.ZIndex`